samples/testapp/src/androidMain/AndroidManifest.xml

```diff
@@ -3,6 +3,12 @@
 
     <uses-permission android:name="android.permission.INTERNET"/>
 
+    <!-- For QR scanning -->
+    <uses-feature android:name="android.hardware.camera"/>
+    <uses-feature android:name="android.hardware.camera.autofocus"/>
+    <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-permission android:name="android.permission.FLASHLIGHT"/>
```

Comment:
Flashlight permission isn't listed in EasyQRScan's list of manifest requirements. And its sample app doesn't list it. Do we need it?

Reply:
We do not.